src/module_wrap.cc

```diff
@@ -244,7 +243,7 @@ MaybeLocal<Module> ModuleWrap::ResolveCallback(Local<Context> context,
   }
 
   Local<Promise> resolve_promise =
-      dependent->resolve_cache_[specifier_std]->Get(iso);
+      dependent->resolve_cache_[specifier_std].Get(iso);
```

Comment:
This habit of using `iso` when the rest of the code base consistently uses `isolate`... urgh.

Reply:
whats the problem? / there are not style docs saying to use `isolate`?

Reply:
That doesn't mean we shouldn't try to keep things consistent.

Reply:
@Fishrock123 I agree consistency is nice, but every pull request I do seems to have style errors and there is no guide; I will continue to bring this up as long as I see comments like "urgh." for innocent mistakes.

Reply:
Have to agree with @bmeck here. There really ought to be a style guide that makes these things more obvious.

Reply:
Not everything needs to be spelled out.  When you start making changes to a code base you're not familiar with, you first peek around to get a feel of the house style.  10 minutes of browsing would show that we use `isolate` everywhere and `iso` nowhere.

Reply:
@bnoordhuis you have even in the past called PRs of mine stupid. That line of reasoning and interactions like the one above make me want things spelled out to avoid such behavior.

Reply:
If it's any consolation to you, I'm also disappointed none of the reviewers called it out.

Reply:
It is not. Mistakes are made by everyone and we should work together, not bring pessimism into things if it can be avoided.

Reply:
Fwiw I was pretty sure I pushed commits to the original PR that did that particular style change, because I knew it wasn’t obvious and I think @bmeck said he preferred that to an endless list of style nits? These *probably* just got lost during some force push or sth.